features/step_definitions/emails.rb

```diff
@@ -23,3 +23,9 @@ def self.get_email(message_type, email_address)
 Then /^I click the link in that email$/ do
   page.visit(URI.extract(@email_text).select { |i| i.start_with?('http') } [0])
 end
+
+Given /^I have an email address$/ do
```

Comment:
Maybe this should be "I have an email address with an allowed buyer email domain" to make it clearer?

Reply:
Good idea to signify that it's a buyer-allowed email address, I will change it now.